cluster-provision/k8s/1.20/conformance.json

```diff
@@ -1 +1,88 @@
-{"Description":"DEFAULT","UUID":"c3bc7d76-6ce8-4c8a-8bcb-5c7ae5fb22a3","Version":"v0.19.0","ResultsDir":"/tmp/sonobuoy","Resources":["apiservices","certificatesigningrequests","clusterrolebindings","clusterroles","componentstatuses","configmaps","controllerrevisions","cronjobs","customresourcedefinitions","daemonsets","deployments","endpoints","ingresses","jobs","leases","limitranges","mutatingwebhookconfigurations","namespaces","networkpolicies","nodes","persistentvolumeclaims","persistentvolumes","poddisruptionbudgets","pods","podlogs","podsecuritypolicies","podtemplates","priorityclasses","replicasets","replicationcontrollers","resourcequotas","rolebindings","roles","servergroups","serverversion","serviceaccounts","services","statefulsets","storageclasses","validatingwebhookconfigurations","volumeattachments"],"Filters":{"Namespaces":".*","LabelSelector":""},"Limits":{"PodLogs":{"Namespaces":"","SonobuoyNamespace":true,"FieldSelectors":[],"LabelSelector":"","Previous":false,"SinceSeconds":null,"SinceTime":null,"Timestamps":false,"TailLines":null,"LimitBytes":null,"LimitSize":"","LimitTime":""}},"QPS":30,"Burst":50,"Server":{"bindaddress":"0.0.0.0","bindport":8080,"advertiseaddress":"","timeoutseconds":10800},"Plugins":null,"PluginSearchPath":["./plugins.d","/etc/sonobuoy/plugins.d","~/sonobuoy/plugins.d"],"Namespace":"sonobuoy","WorkerImage":"sonobuoy/sonobuoy:v0.19.0","ImagePullPolicy":"IfNotPresent","ImagePullSecrets":"","ProgressUpdatesPort":"8099"}
+{
+  "Description": "DEFAULT",
+  "UUID": "c3bc7d76-6ce8-4c8a-8bcb-5c7ae5fb22a3",
+  "Version": "v0.19.0",
```

Comment:
There is no need to bump this one as well?
I guess I miss the relation between the version in this manifest and the one in the conformance.sh script.

Reply:
This change has to be done at different PR, we want the k8s-1.21 provider.

Reply:
Oh, interesting catch, thanks for that! I thought I had updated this already, as I had seen this also, will take another look.

Reply:
> There is no need to bump this one as well?
> I guess I miss the relation between the version in this manifest and the one in the conformance.sh script.

The version in the script is always the latest available at the time of the update. IIUC the version in the conformance config is the one that matters when executing the conformance tests.